README.md

```diff
@@ -24,6 +28,17 @@ To only access metrics related to CPU 2, access http://localhost:8082/node?cpu=2
 Further there is a Prometheus at http://localhost:9090/ that only accesses node metrics that are related to CPU 2.
 
 
+### KinD
+
+To test the `exporter-filterproxy` in Kubernetes using the Kubernetes service discovery you can run:
+
+```
+make kind
+```
+
+This will start a local kind cluster and deploy a `exporter-filterproxy` that proxies the metrics of the CoreDNS service.
+
+
```

Comment:
This will be extended to explain it a bit better once HTTP service discovery is implemented